pkg/resource/properties.go

```diff
@@ -546,3 +570,6 @@ func HasSig(obj PropertyMap, match string) bool {
 
 // SecretSig is the unique secret signature.
 const SecretSig = "1b47061264138c4ac30d75fd1eb44270"
+
+// ResourceSig is the unique resource signature.
+const ResourceSig = "5cf8f73096256a8f31e491e813e4eb8e"
```

Comment:
curious how you generated this.

Reply:
If I recall correctly - `openssl rand -hex 16`.